metadata-ingestion/tests/integration/business-glossary/business_glossary.yml

```diff
@@ -26,6 +26,9 @@ nodes:
         description: Highly Confidential Data
         custom_properties:
           is_confidential: true
+        domains:
+          - en.wikipedia.org
```

Comment:
this is not same concept as web domain- a better example would be `Marketing`

Reply:
done